recipes/cap3/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: cap3
+  version: '10.2011'
+
+source:
+  fn: cap3.linux.x86_64.tar # [linux]
+  url: http://seq.cs.iastate.edu/CAP3/cap3.linux.x86_64.tar # [linux]  
```

Comment:
Ok, I will try to get in touch in touch with them as soon as I'm back to work tomorrow. But I already tried to contact one of them some days ago and I did not have any answer.